### ISSUE TITLE
Can it be used without fixed nodes?

### ISSUE THREAD

**Issue by alwaysqi1121** [uqLab/stedy#14]
I noticed that I have to specify fixed nodes in the simulation code,

**I try to delete all code related to fixed nodes**,but if I didn't specify fixed nodes,it will remind me:

`Optimization failed. Please check input parameters again.`

**Comment by vaishnavtv** [comment]
If you don't want any fixed nodes in your simulation, you can turn them off by placing zeros everywhere in a matrix of the same size as N, i.e., 
```matlab
fixedNodes = zeros(size(N))
```

**Comment by alwaysqi1121** [comment]
Thanks for your reply.
I tried what you said,and I changed the code in `Example\Ball\Ball.m`
I modified the code like this(I commented out the code on the setting of fixed points):

`fixedNodes = zeros(size(N));`
`% fixedNodes(:,1) = [1 1 1]';`
`% fixedNodes(:,2) = [1 1 1]';`
`% fixedNodes(:,9) = [1 1 1]';`
But then , it showed that :
`LP preprocessing removed 0 inequalities, 12 equalities,
13 variables, and 51 non-zero elements.

 Iter      Time            Fval  Primal Infeas    Dual Infeas  
    0     0.001    0.000000e+00   4.651893e+01   0.000000e+00  
   17     0.001    0.000000e+00   1.238909e+03   0.000000e+00  
   18     0.001    0.000000e+00   1.343755e+03   0.000000e+00  

No feasible solution found.

Linprog stopped because no point satisfies the constraints.`

No feasible solution found.

I think it means :the initial force density cannot be obtained...and then, the code will stop,

(1) Why is it necessary to calculate the initial force density before the next operation?
(2) And，how to judge the force density in the cables can balance the whole system?
(3) Do I need to recalculate the force density in the cables every time the coordinates of the nodes changed?

Thanks.
And I really hope to get your reply!
